zerver/lib/digest.py

```diff
@@ -230,11 +230,10 @@ def handle_digest_email(user_profile_id: int, cutoff: float,
         user_profile, [pm.message for pm in pms[:pms_limit]])
     context['remaining_unread_pms_count'] = max(0, len(pms) - pms_limit)
 
-    home_view_recipients = [sub.recipient for sub in
-                            Subscription.objects.filter(
-                                user_profile=user_profile,
-                                active=True,
-                                in_home_view=True)]
+    home_view_recipients = Subscription.objects.filter(
+        user_profile=user_profile,
+        active=True,
+        in_home_view=True).values_list('recipient', flat=True)
```

Comment:
I usually spell this as `recipient_id`, just to be clear it's integers and not objects that we want.

Reply:
Fixed, thanks!